.github/workflows/frontend_tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Frontend tests
+
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  tests:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
What's the idea behind 3 versions here? Shouldn't it (only) be the version used here https://github.com/research-software-directory/research-software-directory/blob/f5861de2c4b5373300110fc0dc08febb505a9709/frontend/Dockerfile#L1 ?

Reply:
using only Python 3.6